test/manual/keychain_provider/stop

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# load the environment with the test config
+source test_config
+
+# delete the password if it exists, and swallow errors if not
```

Comment:
Ideally, we only want to swallow the error if the error code represents "cant delete bc secret not found" but if there was some other error you would want to let the user know because there's still a random test pw in their keychain.

I looked up `man security` and couldn't find the error codes explained so not sure if this is possible but would worth figuring out if it is.